distributed/shuffle/tests/test_shuffle.py

```diff
@@ -144,12 +144,12 @@ async def test_basic_cudf_support(c, s, a, b):
         dtypes={"x": float, "y": float},
         freq="10 s",
     ).to_backend("cudf")
-    out = dd.shuffle.shuffle(df, "x", shuffle="p2p")
-    assert out.npartitions == df.npartitions
-    x, y = c.compute([df.x.size, out.x.size])
-    x = await x
-    y = await y
-    assert x == y
+    shuffled = dd.shuffle.shuffle(df, "x", shuffle="p2p")
+    assert shuffled.npartitions == df.npartitions
+
+    result, expected = c.compute([shuffled, df])
+    result, expected = await c.gather([result, expected])
```

Comment:
First of these lines is redundant without checking for equality, no?

or actually no, I am confused compute brings the "complete" thing to the host, what does the gather then do?

Reply:
These tests are run in async mode, so we still have to await the results.

Reply:
Ah, maybe `c.gather([shuffled, df], sync=True)` or `result, expected = await c.gather(c.compute([shuffled, df]))`?